File: trunk/Templates/Examples/Xml/CSharp/output/Product.cs
using System;

namespace Northwind.DAL
{
	/// <summary>
	/// This object represents the properties and methods of a Product.
	/// </summary>
	public class Product
	{
		private Int32 _productID;
		private String _productName;
		private Int32 _supplierID;
		private Int32 _categoryID;
		private String _quantityPerUnit;
		private Decimal _unitPrice;
		private Int16 _unitsInStock;
		private Int16 _unitsOnOrder;
		private Int16 _reorderLevel;
		private Boolean _discontinued;
		
		public Product()
		{
		}
		
		#region Custom - Methods
		// Insert custom methods in here so that they are preserved during re-generation.
		#endregion
	}
}

File: branches/5.x/Templates/Examples/Xml/VisualBasic/output/Product.vb
Imports System

Namespace Northwind.DAL
	''' <summary>
	''' This object represents the properties and methods of a Product.
	''' </summary>
	Public Class Product
		Private _productID As Int32
		Private _productName As String
		Private _supplierID As Int32
		Private _categoryID As Int32
		Private _quantityPerUnit As String
		Private _unitPrice As Decimal
		Private _unitsInStock As Int16
		Private _unitsOnOrder As Int16
		Private _reorderLevel As Int16
		Private _discontinued As Boolean

		Public Sub New()
		End Sub

		#region Custom - Methods
		' Insert custom methods in here so that they are preserved during re-generation.
#End Region
	End Class
End Namespace

